packages/llvm_stage1.rb

```diff
@@ -0,0 +1,153 @@
+        i686: 'https://dl.bintray.com/chromebrew/chromebrew/llvm_stage1-11.1.0-rc1-chromeos-i686.tar.xz',
+      x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/llvm_stage1-11.1.0-rc1-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+     aarch64: '863dd7cae43abef88dfe25c60e0dda1f8d59756f8c8fbb111989a3669dca0a16',
+      armv7l: '863dd7cae43abef88dfe25c60e0dda1f8d59756f8c8fbb111989a3669dca0a16',
+        i686: '84a5a2a381af8c789fcb895d1ae62dc1932b3ad1ad423e8351e37aa9be5e2490',
+      x86_64: 'ddb9e88444729174b0c6f4de883910ce78e8fa489e0831acdf92daaf4561e9af',
+  })
+  
+  # llvm_stage1 is compiled with gcc, without -flto
+  # Then in stage 2 (llvm.rb) everything is compiled with clang built in stage 1, with -flto=thin
+  # FYI LLVM 11.1.0 is a tiny rebuild of 11.0.1 fixing ABI compatibility with LLVM 10.0.0 & LLVM 12
+
+  depends_on 'ocaml' => :build
+  depends_on 'libedit'
+  depends_on 'libtirpc'
+  depends_on 'swig'
+  depends_on 'pygments' => :build
+  depends_on 'ccache' => :build
+
+  case ARCH
+  when 'aarch64','armv7l'
+    #LLVM_TARGETS_TO_BUILD = 'ARM;AArch64;AMDGPU'
+    @ARCH_C_FLAGS = '-march=armv7-a -mfloat-abi=hard'
+    @ARCH_CXX_FLAGS = '-march=armv7-a -mfloat-abi=hard'
+    LLVM_PROJECTS_TO_BUILD = 'clang;clang-tools-extra;libcxx;libcxxabi;lld'
+  when 'i686','x86_64'
+    #LLVM_TARGETS_TO_BUILD = 'X86;AMDGPU'
+    @ARCH_C_FLAGS = '-fPIC'
+    @ARCH_CXX_FLAGS = '-fPIC'
+    LLVM_PROJECTS_TO_BUILD = 'clang;clang-tools-extra;libcxx;libcxxabi;libunwind;lld'
+  end
+  
+  @ARCH_C_LTO_FLAGS = "#{@ARCH_C_FLAGS} -flto"
+  @ARCH_CXX_LTO_FLAGS = "#{@ARCH_CXX_FLAGS} -flto"
+  
+  # Using Targets 'all' because otherwise mesa complains.
+  # This may be patched upstream as per 
+  # https://reviews.llvm.org/rG1de56d6d13c083c996dfd44a32041dacae037d66
+  LLVM_TARGETS_TO_BUILD = 'all'
+  LLVM_VERSION = version.split("-")[0]
+  BINUTILS_BRANCH = 'gdb-10.1-release'
+
+  def self.build
+    ############################################################
+    puts "Building LLVM Targets: #{LLVM_TARGETS_TO_BUILD}".lightgreen
+    puts "Building LLVM Projects: #{LLVM_PROJECTS_TO_BUILD}".lightgreen
+    ############################################################
+    ############################################################
+    puts "Setting compile to use python3".lightgreen
+    ############################################################
+    system "grep -rl '#!.*python' | xargs sed -i '1s/python$/python3/'"
+    
+    ############################################################
+    puts "Downloading binutils #{BINUTILS_BRANCH} src to enable gold plugin build".lightgreen
+    ############################################################
+    system "git config --global advice.detachedHead false"
+    # As per https://github.com/SVF-tools/SVF/wiki/Install-LLVM-Gold-Plugin-on-Ubuntu
+    system "git clone --depth 1 --branch #{BINUTILS_BRANCH} git://sourceware.org/git/binutils-gdb.git binutils"
```

Comment:
Can we use the binutils package binaries and get rid of this?

Reply:
done